src/audit_report.rs

```diff
@@ -112,6 +124,20 @@ impl AuditReport {
     pub(crate) fn len(&self) -> usize {
         self.records.len()
     }
+
+    pub(crate) fn to_audit_digest(&self) -> AuditDigest {
+        let mut audit_digests = Vec::new();
+
+        for record in &self.records {
+            audit_digests.push(AuditDigestRecord {
+                package: Some(record.package.to_string()),
+                vuln_ids: record.vuln_ids.clone(),
+                vuln_infos: record.vuln_infos.clone(),
+            });
+        }
+
+        audit_digests
+    }
```

Comment:
The reason we have `ValidationDigest` is because we need to simplify things to Strings so that the JSON is simpler and more compact. Do we need to do this for the `AuditReport`? If it only contains strings, can we just serialize the `AuditReport` directly and not use this derived strcut.